pom.xml

```diff
@@ -12,7 +12,7 @@
     <artifactId>gobblin-wmf</artifactId>
```

Comment:
Now that we cleaned up all those unused dependencies, we might be able to remove some of the exclusions in the configuration of the [duplicate-finder-maven-plugin](https://github.com/basepom/duplicate-finder-maven-plugin) (see below in the <pluginManagement/> section.

Reply:
Gehel sez .... do this in another PR